packages/core/src/lib/lit-core.ts

```diff
@@ -943,6 +1003,11 @@ export class LitCore {
       body: JSON.stringify(data),
     };
 
+    const requestTimeout = timeout || this.config.nodeRequestTimeout;
+    if (requestTimeout) {
+      req.signal = AbortSignal.timeout(requestTimeout);
+    }
+
     return sendRequest(url, req, requestId);
```

Comment:
should this be `await`?

Reply:
The function is `async` so it is not needed

Reply:
ah you're right